format.cc

```diff
@@ -425,7 +419,7 @@ FMT_FUNC void fmt::SystemError::init(
 }
 
 template <typename T>
-int fmt::internal::CharTraits<char>::format_float(
+FMT_EXPORT int fmt::internal::CharTraits<char>::format_float(
```

Comment:
Is it necessary to use `__declspec(dllexport)` both on declarations and definitions? If not I suggest using it on the header declarations only.